consul/error.go

```diff
@@ -27,10 +27,14 @@ var (
 	ErrMeshConfigCode               = "1005"
 	ErrConsulCoreComponentFailCode  = "1006"
 	ErrParseConsulCoreComponentCode = "1007"
-	ErrParseOAMComponent            = errors.New(ErrParseOAMComponentCode, errors.Alert, []string{"error parsing the component"}, []string{"Error occurred while prasing application component in the OAM request made"}, []string{"Invalid OAM component passed in OAM request"}, []string{"Check if your request has vaild OAM components"})
-	// ErrParseOAMConfig represents the error which is
-	// generated during the OAM configuration parsing
-	ErrParseOAMConfig = errors.New(ErrParseOAMConfigCode, errors.Alert, []string{"error parsing the configuration"}, []string{"Error occurred while prasing component config in the OAM request made"}, []string{"Invalid OAM config passed in OAM request"}, []string{"Check if your request has vaild OAM config"})
+	
+	// ErrParseOAMComponent represents the error which is
+ 	// generated during the OAM component parsing
+ 	ErrParseOAMComponent = errors.New(ErrParseOAMComponentCode, errors.Alert, []string{"error parsing the component"}, []string{"Error occurred while parsing application component in the OAM request made by Meshery server"}, []string{"Could not unmarshall configuration component received via ProcessOAM gRPC call into a valid Component struct"}, []string{"Check if Meshery Server is creating valid component for ProcessOAM gRPC call. This error should never happen and can be reported as a bug in Meshery Server. Also check if Meshery Server and adapters are referring to same component struct provided in MeshKit."})
```

Comment:
Capitalize Meshery Server, please. 
Add comma after "Also".

Reply:
Oh no, need to make these changes on other PR too